### DISCUSSION TITLE
Split temperature support

### DISCUSSION THREAD

**Discussion by kfox1111** [grafana/grafana#38403] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

It would be cool if you could select the native temperature (F or C) that the data is in, and have the option for the graph to display the same data with F on the left and C on the right and when hovering over the data, display the data in both formats.

**Why is this needed**:
Some of our users are familiar with C, and some are familiar with F. Being able to easily present the same data both ways would make the graphs much more usable.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
